metadata/Cadastre/land_ownership

```diff
@@ -0,0 +1,89 @@
+# Title
+
+Utah Land Ownership
+
+## ID
+
+e9eba9ba-3950-4b99-89d2-b1d9282133b6
+
+## Brief Summary
+
+The official State of Utah School and Institutional Trust Lands Administration (SITLA) surface land ownership data.
+
+## Summary
+
+Maintained by a joint effort between SITLA, the BLM, and UGRC, this dataset represents current land ownership in the state of Utah, categorized as federal, state, private, or tribal land. In the cases of federal, state, or private land, this dataset specifies the organization or agency that claims stewardship over that area. However, in the case of private land, this dataset does not identify the specific landowner.
+
+## Description
+
+### What is the dataset?
+
+This dataset depicts land ownership status and areas of responsibility for surface lands in the State of Utah. Land ownership is represented as polygons at a scale of 1:24,000.
+
+Standard colors are recommended when displaying this feature class. A layer file containing the standard color scheme is available by contacting SITLA GIS staff directly at the GIS staff from SITLA at [TLA-GIS@utah.gov] | 801-538-5100, or visiting [https://trustlands.utah.gov].
+
+<!--- Did some digging on the SITLA website and did not find the standard color scheme mentioned, so I believe users will need to contact SITLA directly if they are interested in using it. -->
+
+### What is the purpose of the dataset?
+
+Land ownership data is used by many government and private entities to demarcate boundaries, provide evidence in legal disputes, and display administrative jurisdictions for agencies such as the Bureau of Land Management and the US Forest Service. This dataset is also a helpful resource for public recreation, news outlets, and other interested parties.
```

Comment:
Is it worth mentioning analysis?

Reply:
Absolutely! I can add it in. Other than mentioning it is generally used for analysis, are there specific examples you'd like to see included?